submissions/fabunmiTolulope/javascript/conditionals.js

```diff
@@ -0,0 +1,24 @@
+// The positive difference between 7 and 123 is 116 so this program will print numbers between 1 and 116
+let w = 7;
+let y = 123;
+z = y-w;
```

Comment:
It is advisable to always use well descriptive variable names. things like `y, w, x, a, b` and so on are not conventional ways of naming things. imagine entering a codebase and you start seeing names like that. You will find it difficult reading that code. For me, I would have said `let num1 = 7; let num2 = 123;` then `let diff = num2 - num1`

Reply:
Thanks for the observation, I have used more descriptive words for my variables to make it more readable.